js-basics/react-basics_hello-world/src/App.js

```diff
@@ -2,5 +2,9 @@ import React from "react";
 import "./styles.css";
 
 export default function App() {
-  return <div>Say hello...</div>;
+  return <h1>Hello world!</h1>;
 }
```

Comment:
Hey :) dont know if your code is correct
I have done it like this, 
than you don't have to touch the index.js:

export default function App() {
  return <HelloWorld />;
}
function HelloWorld() {
  return <h1>Hello world!</h1>;
}

Reply:
everything works, but at first I didn't quite understand correctly.  I have already completed the following tasks correctly.Thanx😊